docs/tasks/administer-cluster/dns-horizontal-autoscaler.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kube-dns-autoscaler
+  namespace: kube-system
+  labels:
+    k8s-app: kube-dns-autoscaler
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-dns-autoscaler
+    spec:
+      containers:
+      - name: autoscaler
+        image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:1.0.0
+        resources:
+            requests:
+                cpu: "20m"
+                memory: "10Mi"
+        command:
+          - /cluster-proportional-autoscaler
+          - --namespace=kube-system
+          - --configmap=kube-dns-autoscaler
+          - --mode=linear
+          - --target={SCALE_TARGET}
```

Comment:
Unless there's a reason to use braces for this placeholder, I would suggest angle brackets.
--target=<SCALE_TARGET>
That would be in line with our style guide. If we change to angle brackets here, we also need to make the change in dns-horizontal-autoscaling.md.